### ISSUE TITLE
ElasticSearch Query Parsing when searching for / and probably for (+ - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /)

### ISSUE THREAD

**Issue by tkurze** [opencast/opencast#6298]
**Describe the bug**

When entering / or any(?) other string containing '/' (such as "WS24/25") an error is logged (org.elasticsearch.client.ResponseException, "Failed to parse query [*/*]") in the opencast.log. In older versions of Opencast (13.7) the request to Elasticsearch is repeated very often (potentially an unlimited number of times?) causing the log file to grow extremely in size (potentially to take up all space on the disk...) 
The current version (16.6) still repeats the request as long as a problematic character is present in the search box (in the admin UI), but stops once the textbox is empty.

**To Reproduce**

1. Go to the Opencast Admin Interface and login 
2. Go to Events (or Series) 
3. enter / in the search box

![image](https://github.com/user-attachments/assets/450e9f1d-1928-4daa-8e28-97abf0311d8c)

**Expected behavior**

I'm not an expert on Elasticsearch / opensearch / etc. but a query string should be escaped correctly in order to avoid an illegal query at all, thus avoiding any error in the log. If (for whatever reason) a query fails, the failure should bubble to the UI and the UI (or any other concerned component) should stop repeating the request.
(+ - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /)

**Stack trace**

If applicable, add a stack trace to help explain your problem.
```
Caused by: org.elasticsearch.ElasticsearchStatusException: Elasticsearch exception [type=search_phase_execution_exception, reason=all shards failed]
opencast-1    | 	at org.elasticsearch.rest.BytesRestResponse.errorFromXContent(BytesRestResponse.java:187) ~[?:?]
opencast-1    | 	at org.elasticsearch.client.RestHighLevelClient.parseEntity(RestHighLevelClient.java:1911) ~[?:?]
opencast-1    | 	at org.elasticsearch.client.RestHighLevelClient.parseResponseException(RestHighLevelClient.java:1888) ~[?:?]
opencast-1    | 	at org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1645) ~[?:?]
opencast-1    | 	at org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1602) ~[?:?]
opencast-1    | 	at org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1572) ~[?:?]
opencast-1    | 	at org.elasticsearch.client.RestHighLevelClient.search(RestHighLevelClient.java:1088) ~[?:?]
opencast-1    | 	at org.opencastproject.elasticsearch.impl.AbstractElasticsearchIndex.executeQuery(AbstractElasticsearchIndex.java:661) ~[?:?]
opencast-1    | 	at org.opencastproject.elasticsearch.index.ElasticsearchIndex.getByQuery(ElasticsearchIndex.java:743) ~[?:?]
opencast-1    | 	... 144 more
opencast-1    | 	Suppressed: org.elasticsearch.client.ResponseException: method [POST], host [http://opensearch:9200], URI [/opencast_event/_search?typed_keys=true&max_concurrent_shard_requests=5&ignore_unavailable=false&expand_wildcards=open&allow_no_indices=true&preference=_local&ignore_throttled=true&search_type=query_then_fetch&batched_reduce_size=512&ccs_minimize_roundtrips=true], status line [HTTP/1.1 400 Bad Request]
opencast-1    | {"error":{"root_cause":[{"type":"query_shard_exception","reason":"Failed to parse query [*/*]","index":"opencast_event","index_uuid":"5MYuifSgQMClgGDcCZcMIQ"}],"type":"search_phase_execution_exception","reason":"all shards failed","phase":"query","grouped":true,"failed_shards":[{"shard":0,"index":"opencast_event","node":"SR3EDD91Tk-ZsylTR7iS4g","reason":{"type":"query_shard_exception","reason":"Failed to parse query [*/*]","index":"opencast_event","index_uuid":"5MYuifSgQMClgGDcCZcMIQ","caused_by":{"type":"parse_exception","reason":"Cannot parse '*/*': Lexical error at line 1, column 4.  Encountered: <EOF> after : \"/*\"","caused_by":{"type":"token_mgr_error","reason":"Lexical error at line 1, column 4.  Encountered: <EOF> after : \"/*\""}}}}]},"status":400}
opencast-1    | 		at org.elasticsearch.client.RestClient.convertResponse(RestClient.java:326) ~[?:?]
opencast-1    | 		at org.elasticsearch.client.RestClient.performRequest(RestClient.java:296) ~[?:?]
opencast-1    | 		at org.elasticsearch.client.RestClient.performRequest(RestClient.java:270) ~[?:?]
opencast-1    | 		at org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1632) ~[?:?]
opencast-1    | 		at org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1602) ~[?:?]
opencast-1    | 		at org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1572) ~[?:?]
opencast-1    | 		at org.elasticsearch.client.RestHighLevelClient.search(RestHighLevelClient.java:1088) ~[?:?]
opencast-1    | 		at org.opencastproject.elasticsearch.impl.AbstractElasticsearchIndex.executeQuery(AbstractElasticsearchIndex.java:661) ~[?:?]
opencast-1    | 		at org.opencastproject.elasticsearch.index.ElasticsearchIndex.getByQuery(ElasticsearchIndex.java:743) ~[?:?]
opencast-1    | 		at org.opencastproject.elasticsearch.index.ElasticsearchIndex.getByQuery(ElasticsearchIndex.java:720) ~[?:?]
opencast-1    | 		at org.opencastproject.adminui.endpoint.AbstractEventEndpoint.getEvents(AbstractEventEndpoint.java:2655) ~[?:?]
opencast-1    | 		at jdk.internal.reflect.GeneratedMethodAccessor42.invoke(Unknown Source) ~[?:?]
opencast-1    | 		at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
opencast-1    | 		at java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
opencast-1    | 		at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:179) ~[?:?]
opencast-1    | 		at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96) ~[?:?]
opencast-1    | 		at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:201) ~[?:?]
opencast-1    | 		at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:104) ~[?:?]
opencast-1    | 		at org.apache.cxf.interceptor.ServiceInvokerInterceptor$1.run(ServiceInvokerInterceptor.java:59) ~[?:?]
opencast-1    | 		at org.apache.cxf.interceptor.ServiceInvokerInterceptor.handleMessage(ServiceInvokerInterceptor.java:96) ~[?:?]
opencast-1    | 		at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:307) ~[?:?]
opencast-1    | 		at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121) ~[?:?]
opencast-1    | 		at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:265) ~[?:?]
opencast-1    | 		at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234) ~[?:?]
opencast-1    | 		at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208) ~[?:?]
opencast-1    | 		at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160) ~[?:?]
opencast-1    | 		at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:225) ~[?:?]
opencast-1    | 		at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:304) ~[?:?]
opencast-1    | 		at org.opencastproject.kernel.rest.RestPublisher$RestServlet.handleRequest(RestPublisher.java:582) ~[?:?]
opencast-1    | 		at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doGet(AbstractHTTPServlet.java:222) ~[?:?]
opencast-1    | 		at javax.servlet.http.HttpServlet.service(HttpServlet.java:497) ~[!/:4.0.4]
opencast-1    | 		at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:279) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedServlet.service(OsgiInitializedServlet.java:102) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHolder$NotAsync.service(ServletHolder.java:1459) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1656) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.opencastproject.kernel.rest.JsonpFilter.doFilter(JsonpFilter.java:130) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedFilter.doFilter(OsgiInitializedFilter.java:176) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebFilterHolder.doFilter(PaxWebFilterHolder.java:208) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.opencastproject.kernel.filter.proxy.TransparentProxyFilter.doFilter(TransparentProxyFilter.java:80) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedFilter.doFilter(OsgiInitializedFilter.java:176) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebFilterHolder.doFilter(PaxWebFilterHolder.java:208) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.opencastproject.security.urlsigning.filter.UrlSigningFilter.doFilter(UrlSigningFilter.java:101) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedFilter.doFilter(OsgiInitializedFilter.java:176) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebFilterHolder.doFilter(PaxWebFilterHolder.java:208) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.opencastproject.kernel.security.RemoteUserAndOrganizationFilter.doFilter(RemoteUserAndOrganizationFilter.java:265) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedFilter.doFilter(OsgiInitializedFilter.java:176) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebFilterHolder.doFilter(PaxWebFilterHolder.java:208) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.opencastproject.kernel.rest.CurrentJobFilter.doFilter(CurrentJobFilter.java:107) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedFilter.doFilter(OsgiInitializedFilter.java:176) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebFilterHolder.doFilter(PaxWebFilterHolder.java:208) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:330) ~[?:?]
opencast-1    | 		at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:118) ~[?:?]
opencast-1    | 		at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:84) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.opencastproject.kernel.security.AsyncTimeoutRedirectFilter.doFilter(AsyncTimeoutRedirectFilter.java:60) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:113) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:103) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:113) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter.doFilter(RememberMeAuthenticationFilter.java:146) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:54) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:45) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:98) ~[?:?]
opencast-1    | 		at org.springframework.security.web.authentication.www.BasicAuthenticationFilter.doFilter(BasicAuthenticationFilter.java:150) ~[?:?]
opencast-1    | 		at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:102) ~[?:?]
opencast-1    | 		at org.springframework.security.web.authentication.www.DigestAuthenticationFilter.doFilter(DigestAuthenticationFilter.java:115) ~[?:?]
opencast-1    | 		at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:102) ~[?:?]
opencast-1    | 		at org.springframework.web.filter.CompositeFilter.doFilter(CompositeFilter.java:82) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.authentication.ui.DefaultLoginPageGeneratingFilter.doFilter(DefaultLoginPageGeneratingFilter.java:91) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:183) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:98) ~[?:?]
opencast-1    | 		at org.springframework.web.filter.CompositeFilter.doFilter(CompositeFilter.java:82) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:105) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:87) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:192) ~[?:?]
opencast-1    | 		at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160) ~[?:?]
opencast-1    | 		at org.opencastproject.kernel.security.SecurityFilter.doFilter(SecurityFilter.java:141) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedFilter.doFilter(OsgiInitializedFilter.java:176) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebFilterHolder.doFilter(PaxWebFilterHolder.java:208) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.opencastproject.kernel.security.OrganizationFilter.doFilter(OrganizationFilter.java:154) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedFilter.doFilter(OsgiInitializedFilter.java:176) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebFilterHolder.doFilter(PaxWebFilterHolder.java:208) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.opencastproject.kernel.filter.https.HttpsFilter.doFilter(HttpsFilter.java:90) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedFilter.doFilter(OsgiInitializedFilter.java:176) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebFilterHolder.doFilter(PaxWebFilterHolder.java:208) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.opencastproject.kernel.rest.CleanSessionsFilter.doFilter(CleanSessionsFilter.java:104) ~[?:?]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiInitializedFilter.doFilter(OsgiInitializedFilter.java:176) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:201) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebFilterHolder.doFilter(PaxWebFilterHolder.java:208) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.spi.servlet.OsgiFilterChain.doFilter(OsgiFilterChain.java:113) ~[!/:?]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PaxWebServletHandler.doHandle(PaxWebServletHandler.java:334) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:600) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1440) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:505) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1355) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:191) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.ops4j.pax.web.service.jetty.internal.PrioritizedHandlerCollection.handle(PrioritizedHandlerCollection.java:96) ~[!/:?]
opencast-1    | 		at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.Server.handle(Server.java:516) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:487) ~[!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:732) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:479) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:338) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:315) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883) [!/:9.4.54.v20240208]
opencast-1    | 		at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034) [!/:9.4.54.v20240208]
opencast-1    | 		at java.lang.Thread.run(Thread.java:840) [?:?]
```

**Server environment for OC 13.7**
 - OS: Ubuntu 22.04.5 LTS 
 - Distribution: all opencast components and DB on different nodes (elasticsearch on admin node)
- Installation method: apt packages
 - Version 13.7

**Server environment for OC 16.6**
 - OS: Mac OS 15.1
 - Distribution: allinone
- Installation method: docker-container
 - Version 16.6

**Comment by mtneug** [comment]
Please update to at least 13.10 or 14.4 (see #5033 and #5150).

**Comment by tkurze** [comment]
You are right, the consequences are less important in newer versions (I also tested with 16.6), but it is still a bug not correctly escaping the query string in my opinion. Even in 16.6 if you just search for "/" an error will be logged in opencast.log, stating that the query can't be parsed.

**Comment by gregorydlogan** [comment]
I should hopefully have some news on this next week.

**Comment by lkiesow** [comment]
Fix should be in the 16.7 release tomorrow.

**Comment by lkiesow** [comment]
Fixed in [16.7](https://github.com/opencast/opencast/releases/tag/16.7) and 17.0.
